### PR TITLE
Create 2019-06-25-prestashop-1-7-6-0-rc-release.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Go download and install this RC version as a test store, either on your machine
 
 **Core:**
   - [Module Faceted search version has been bumped to 3.0.5](https://github.com/PrestaShop/PrestaShop/pull/14284) which contains interesting bug fixes
+  - Watermark and Product comments modules have been integrated and can now be installed from the Module Catalog page in the back office ;
```
Comment (colinegin) [inline news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md]: Add also : "- Tips and Updates module (psaddonsconnect) has been updated to the 2.0.0 version"

Comment (ttoine) [reply news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md]: done

File: news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+tags:
+ - version
+ - rc
+ - minor
+ - releases
+ - 1.7.6.x
+---
+
+
+This is the last step before releasing the final 1.7.6.0. It aims at making sure that all regressions found during the beta have been fixed, and that no new ones appeared. If no new regression is reported by June 27, we will make the final version of 1.7.6.0 available.
```
Comment (colinegin) [inline news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md]: If no new regression is reported by July 1st, we will make the final version of 1.7.6.0 available.

File: news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+**Core:**
+  - [Module Faceted search version has been bumped to 3.0.5](https://github.com/PrestaShop/PrestaShop/pull/14284) which contains interesting bug fixes
+  - Watermark and Product comments modules have been integrated and can now be installed from the Module Catalog page in the back office
+  - Tips and Updates module (psaddonsconnect) has been updated to the 2.0.0 version
+  - [We have replaced PHPExcel by PHPSpreadsheet](https://github.com/PrestaShop/PrestaShop/pull/14316) in order to better handle XLS or XLSX files import
+  
+  
+## Changelog
+
+100+ pull requests have been merged since the beta release and 50+ regressions have been fixed. Read the [Changelog](https://github.com/PrestaShop/PrestaShop/releases) for details. A huge thanks to everyone involved in testing and fixing issues during the beta phase, and especially 202 ecommerce, OneDotIT, MartinF who helped us a lot during this period !
```
Comment (colinegin) [inline news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md]: and especially 202 ecommerce, OneDotIT and MartinF who helped us a lot during this period !

File: news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md (hunk 2 above)
Comment (ttoine) [reply news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md]: done

File: news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md (hunk 3 above)
Comment (ttoine) [reply news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md]: done

File: news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+[Download PrestaShop 1.7.6 RC now!](https://www.prestashop.com/en/developers-versions){: .btn .btn-lg .btn-success}
+{: .text-center}
+
+
+## How to help finish the 1.7.6
+Your help will be very appreciated to finish the work on PrestaShop 1.7.6. Even if you are not a developer, your feedback has a lot of value.
+
+You can:
+
+Install it and test that there is no regression compared to the 1.7.5.2 version
```
Comment (matks) [inline news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md:63]: Shouldn't we add some `-` ?
```
- If possible, test with data imported from a real shop
- If you develop modules or themes, test them on this version
- Test the auto-upgrade plugin
```

Comment (ttoine) [reply news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md:63]: Should be ; and .
But we can let it open too.

**Pull request merged**